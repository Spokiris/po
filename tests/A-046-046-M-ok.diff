*** tests/A-046-046-M-ok.out	2023-10-19 17:08:13.000000000 +0100
--- tests/A-046-046-M-ok.outhyp	2023-10-27 19:48:47.323666468 +0100
***************
*** 14,45 ****
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
! 0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|23=SUB(2;1,2)
! 4;2|23=SUB(2;1,2)
! 4;3|23=SUB(2;1,2)
! Menu de Edição
! 1 - Visualizar
! 2 - Inserir
! 3 - Copiar
! 4 - Apagar
! 5 - Cortar
! 6 - Colar
! 7 - Mostrar 'cut buffer'
! 0 - Sair
! Escolha uma opção: Menu Principal
! 1 - Novo
! 2 - Abrir
! 3 - Guardar
! 4 - Menu de Edição
! 5 - Menu de Consultas
! 0 - Sair
! Escolha uma opção: 
\ No newline at end of file
--- 14,17 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
! Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: 
\ No newline at end of file
