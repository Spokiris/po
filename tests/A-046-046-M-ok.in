4
2
4;1:4;3
=SUB(2;1,2)
1
4;1:4;3
0
0
