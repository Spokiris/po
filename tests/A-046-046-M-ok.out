Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 4
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 2
Especifique a gama (startline;startcol:endline;endcol): 4;1:4;3
Insira o conteúdo da célula: =SUB(2;1,2)
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1
Especifique a gama (startline;startcol:endline;endcol): 4;1:4;3
4;1|23=SUB(2;1,2)
4;2|23=SUB(2;1,2)
4;3|23=SUB(2;1,2)
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
